db/migrations/20230619063845_create_event_token_payments_table.sql

```diff
@@ -3,7 +3,7 @@ CREATE TABLE event_token_payments (
   id INT NOT NULL AUTO_INCREMENT,
   user_id INT NOT NULL,
   payment_status_code VARCHAR(30) NOT NULL,
-  information_json VARCHAR(1000) NOT NULL,
+  information_json VARCHAR(2000) NOT NULL,
```

Comment:
- 여기도 기존 파일에 추가하지 말고 새로운 migration 파일을 만들어 주세요.
- JSON 형태가 저장 된다면, 데이터 타입도 json으로 해주세요
```suggestion
  information_json VARCHAR(2000) NOT NULL,
```